server/api/book-list/index.post.ts

```diff
@@ -0,0 +1,21 @@
+export default defineEventHandler(async (event) => {
+  const session = await requireUserSession(event)
+  const userWallet = session.user.evmWallet
+  const body = await readBody(event)
+  const { nftClassId, priceIndex } = body
+  if (!nftClassId) {
+    throw createError({
+      statusCode: 400,
+      statusMessage: 'nftClassId is required in body',
+    })
+  }
+  if (priceIndex === undefined) {
```

Comment:
Not sure if we want to default to `0` in these cases. Also might want to check is valid positive integer